### ISSUE TITLE
Is similar to JSVerbalExpressions

### ISSUE THREAD

**Issue by shreyasminocha** [areknawo/Rex#27]
https://github.com/VerbalExpressions/JSVerbalExpressions

Perhaps add a section to your readme clarifying how your lib is better than similar existing projects.

**Comment by areknawo** [comment]
That's right. It also might be worth it to look for any other more popular alternatives to list. I most likely will take a look at it in the near future when I'll be updating the lib. I'm open to suggestions tho.
